src/Features/UniversalRoutes.php

```diff
@@ -23,12 +23,14 @@ class UniversalRoutes implements Feature
     public function bootstrap(Tenancy $tenancy): void
     {
         foreach (static::$identificationMiddlewares as $middleware) {
-            $middleware::$onFail = function ($exception, $request, $next) {
+            $originalOnFail = $middleware::$onFail;
```

Comment:
I believe this can be `null` sometimes? Which would make the `$originalOnFail()` call result in an error.

Though tests seem to be passing

Reply:
You are right. When the Universal Route feature is enabled, and `onFail` logic is not provided, we get the `Value of type null is not callable` error instead of the `TenantCouldNotBeIdentifiedOnDomainException` exception.